src/routes/transactions/entities/multisig-execution-info.entity.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { AddressInfo } from '../../common/entities/address-info.entity';
+import { ExecutionInfo } from './execution-info.entity';
+
+export class MultisigExecutionInfo extends ExecutionInfo {
+  @ApiProperty()
+  nonce: number;
+  @ApiProperty()
+  confirmationsRequired: number;
+  @ApiProperty()
+  confirmationsSubmitted: number;
+  @ApiPropertyOptional({ type: Array<AddressInfo>, nullable: true })
```

Comment:
I don't think the generic can be applied here (it's expecting an expression) 🤔

Reply:
True, fixed in [d985d6c](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/d985d6cb5043fb4c54881c2cdab35c7007f20306)